<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="59.9780" tests="18" failures="2">
  <testsuite name="Root Suite" timestamp="2022-11-03T15:45:12" tests="0" file="tests/e2e/specs/nobodyTesting.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Test Login" timestamp="2022-11-03T15:45:12" tests="9" time="26.4340" failures="0">
    <testcase name="Test Login Input Fields are there" time="5.8880" classname="Input Fields are there">
    </testcase>
    <testcase name="Test Login Navigationt to Password Reset is here" time="2.7410" classname="Navigationt to Password Reset is here">
    </testcase>
    <testcase name="Test Login login error works Works" time="4.8680" classname="login error works Works">
    </testcase>
    <testcase name="Test Login check if register link works" time="2.2200" classname="check if register link works">
    </testcase>
    <testcase name="Test Login check if password reset link works" time="2.2890" classname="check if password reset link works">
    </testcase>
    <testcase name="Test Login check if redirection works from manager" time="1.6580" classname="check if redirection works from manager">
    </testcase>
    <testcase name="Test Login check if redirection works from customer" time="1.6420" classname="check if redirection works from customer">
    </testcase>
    <testcase name="Test Login check if redirection works from picker" time="2.0720" classname="check if redirection works from picker">
    </testcase>
    <testcase name="Test Login check if redirection works from picker" time="3.0560" classname="check if redirection works from picker">
    </testcase>
  </testsuite>
  <testsuite name="Test Create Account" timestamp="2022-11-03T15:45:42" tests="9" time="33.5440" failures="2">
    <testcase name="Test Create Account Input Fields are there" time="3.2440" classname="Input Fields are there">
    </testcase>
    <testcase name="Test Create Account Navigationt to Password Reset is here" time="2.0060" classname="Navigationt to Password Reset is here">
    </testcase>
    <testcase name="Test Create Account test if inputs matter" time="7.7400" classname="test if inputs matter">
    </testcase>
    <testcase name="Test Create Account test if inputs matter" time="3.6660" classname="test if inputs matter">
    </testcase>
    <testcase name="Test Create Account test if registration works" time="0.0000" classname="test if registration works">
      <failure message="Timed out retrying after 4000ms: Expected to find content: &apos;Du solltest in den folgenden Minuten eine Email mit einem Bestätigungslink&apos; within the selector: &apos;p&apos; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Du solltest in den folgenden Minuten eine Email mit einem Bestätigungslink' within the selector: 'p' but never did.
    at Context.eval (http://localhost:8080/__cypress/tests?p=tests\e2e\specs\nobodyTesting.js:222:8)]]></failure>
    </testcase>
    <testcase name="Test Create Account test if login works" time="7.1740" classname="test if login works">
    </testcase>
    <testcase name="Test Create Account test if password reset without input does not work" time="3.2090" classname="test if password reset without input does not work">
    </testcase>
    <testcase name="Test Create Account test if password reset with false email does not work" time="6.5050" classname="test if password reset with false email does not work">
    </testcase>
    <testcase name="Test Create Account test if password reset works" time="0.0000" classname="test if password reset works">
      <failure message="Timed out retrying after 4000ms: Expected to find content: &apos;Du solltest in den folgenden Minuten eine Email mit einem Link erhalten.&apos; within the selector: &apos;p&apos; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Du solltest in den folgenden Minuten eine Email mit einem Link erhalten.' within the selector: 'p' but never did.
    at Context.eval (http://localhost:8080/__cypress/tests?p=tests\e2e\specs\nobodyTesting.js:257:8)]]></failure>
    </testcase>
  </testsuite>
</testsuites>